src/backend/mod.rs

```diff
@@ -1,11 +1,13 @@
 use event;
 use theme;
 
-// Module is not named `ncurses` to avoir naming conflict
-#[cfg(feature = "ncurses")]
-pub mod curses;
 #[cfg(feature = "termion")]
-pub mod termion;
+mod termion;
+mod curses;
+
+pub use self::curses::*;
+#[cfg(feature = "termion")]
+pub use self::termion::*;
```

Comment:
Now the back-end is at `backend::Concrete`, always.
`lib.rs` contains no more conditionals.